### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nJobes**

### CODE REVIEW

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java [hunk 1]
```diff
@@ -17,6 +18,7 @@
 import com.cst438.service.GradebookService;
 
 @RestController
+@CrossOrigin(origins = {"http://localhost:3000", "https://registerf-cst438.herokuapp.com/"})
```
Comment (jeromeportega) [inline cst438_register/src/main/java/com/cst438/controller/StudentController.java:21]: Ha, pretty cool you were trying to use your deployed database dev instance.

File: cst438_register/src/main/java/com/cst438/domain/StudentDTO.java [hunk 2]
```diff
@@ -6,13 +6,15 @@ public class StudentDTO {
...
 	public String status;
+	public String message;
 	
 	public StudentDTO() {
 		this.student_id = 0;
 		this.name = null;
 		this.email = null;
 		this.status = null;
 		this.statusCode = 0;
+		this.message = "";
```
Comment (jeromeportega) [inline cst438_register/src/main/java/com/cst438/domain/StudentDTO.java:17]: Interesting.  Wouldn't the status code and the message be added on via the Spring Boot framework?  I wonder if the DTO is just the data itself, outside of the response details.

Comment (nJobes) [reply cst438_register/src/main/java/com/cst438/domain/StudentDTO.java:17]: The status code is a field of the Student table. It represents a hold on a student's account. Status code == 0 means all clear, StatusCode !=0 means a hold is present, and Status is a string that would then read something like "Payment overdue" or "Alcohol Awareness". As far as the message unit yes, that could technically be slapped into the HTTP response although maybe a meaningless change at the moment.

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java (hunk 1 above)
Comment (nJobes) [reply cst438_register/src/main/java/com/cst438/controller/StudentController.java:21]: I actually wasn't I believe this is from the original source repo, I assumed it was for testing/grading.

**Comment by jeromeportega** [comment]
Nice, honestly looks great.

**Pull request merged**